src/zjs_ws.json

```diff
@@ -15,7 +15,7 @@
             "CONFIG_NET_STATISTICS=y",
             "CONFIG_NET_BUF=y",
             "CONFIG_NET_PKT_RX_COUNT=5",
-            "CONFIG_NET_PKT_TX_COUNT=2",
+            "CONFIG_NET_PKT_TX_COUNT=5",
             "CONFIG_NET_BUF_RX_COUNT=10",
             "CONFIG_NET_BUF_TX_COUNT=6",
```

Comment:
Really it makes sense for TX and RX to not match though (as in buf count) because we have control over TX but not so much RX.